tests/devapps/MauiApps/MauiAppB2C/MainPage.xaml.cs

```diff
@@ -12,30 +12,33 @@ public MainPage()
 		InitializeComponent();
 	}
 
-	private async void OnSignInClicked(object sender, EventArgs e)
-	{
+    private async void OnSignInClicked(object sender, EventArgs e)
+    {
         try
         {
-            var authResult = await B2CAuthenticationService.Instance.SignInAsync().ConfigureAwait(false);
+            // attempt silent login.
+            // If this is very first time or user has signed out, it will throw MsalUiRequiredException
+            AuthenticationResult result = await PCAWrapperB2C.Instance.AcquireTokenSilentAsync(B2CConstants.Scopes).ConfigureAwait(false);
+
+            // show the IdToken
+            await ShowMessage("AcquireTokenTokenSilent call IdToken", result.IdToken).ConfigureAwait(false);

```

Comment:
instead of showing the token can we show something relevant for the user?

Reply:
I think this will be good enough for the developers. Else it will need parsing etc. If you have any suggestion that does not require parsing, I will be happy to display it. IdToken tells that it can be retrieved.